deployment/ansible/roles/model-my-watershed.tiler/defaults/main.yml

```diff
@@ -1,7 +1,3 @@
 ---
 tiler_home: /opt/tiler
 tiler_deploy_branch: "master"
-
-tiler_log: /var/log/mmw-tiler.log
```

Comment:
Log rotation no longer necessary for this service. Logs are routed directly to `syslog` via `systemd`.